src/ui/ui-kit/FrameList/FrameListItem.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import cn from 'classnames';
+import type { LinkProps } from 'react-router-dom';
+import { UnstyledLink } from '../UnstyledLink';
+import { UnstyledAnchor } from '../UnstyledAnchor';
+import { UnstyledButton } from '../UnstyledButton';
+import * as styles from './styles.module.css';
+
+export const FrameListItem = React.forwardRef(
+  (
+    { className, style, ...props }: React.HTMLAttributes<HTMLDivElement>,
+    ref: React.Ref<HTMLDivElement>
+  ) => {
+    return (
+      <div
+        ref={ref}
+        style={{ color: 'inherit', ...style }}
+        className={cn(styles.item, 'parent-hover', className)}
```

Comment:
I don't think the `.parent-hover` class should be baked into the components. When it's needed, the consumer code can apply it

Reply:
Ok :(